### PR TITLE
fix: order card aligned in mobile view

### LINKED ISSUE #6154: Need little enhancement in mobile view
**Describe the bug**

The font is not responsive in mobile view

**Screenshots**

Screenshot (113)_LI

### PR DESCRIPTION
**Pull request by afzal442**
Fixes #6154 
Screenshot Before
c3

Screenshot After
c2

#### Changes proposed in this pull request:

- properly aligned the order card
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/styles/partials/event-card.scss [hunk 1]
```diff
@@ -48,10 +48,6 @@
 
   .main.content {
     display: block;
-    height: 130px;
+    padding-bottom: 0px;
```
Comment (MeghalBisht) [inline app/styles/partials/event-card.scss:51]: Screenshot from 2021-02-03 22-50-12

Not checked in button should always be at the bottom of the card.

Comment (afzal442) [reply app/styles/partials/event-card.scss:51]: Thanks. Does everything else work fine?

Comment (MeghalBisht) [reply app/styles/partials/event-card.scss:51]: Your PR changes the behaviour of explore page
Screenshot from 2021-02-03 23-03-18

Comment (afzal442) [reply app/styles/partials/event-card.scss:51]: Sorry to see that. Making changes. Thanks.

Comment (afzal442) [reply app/styles/partials/event-card.scss]: Plz review it now. Thanks.

File: app/styles/partials/event-card.scss [hunk 2]
```diff
@@ -54,4 +54,4 @@
 
 .order.card {
   height: 100px !important;
```
Comment (MeghalBisht) [inline app/styles/partials/event-card.scss:56]: Try as min-height: 100px 
Screenshot from 2021-02-04 15-31-18

Comment (afzal442) [reply app/styles/partials/event-card.scss:56]: Ohk got it. Let me try.

Comment (afzal442) [reply app/styles/partials/event-card.scss:56]: But don't mind why did u change ui grid to ui menu and like mobile only row??

Comment (afzal442) [reply app/styles/partials/event-card.scss:56]: Otherwise it's working fine. Plz let me know if I am wrong here.

Comment (MeghalBisht) [reply app/styles/partials/event-card.scss:56]: I did not change any style, its your deploy link that is opened.

Comment (afzal442) [reply app/styles/partials/event-card.scss:56]: Alright. I have got this as screenshot below. Have a look
c1

Comment (MeghalBisht) [reply app/styles/partials/event-card.scss:56]: Try reducing the width further

Comment (afzal442) [reply app/styles/partials/event-card.scss:56]: Thanks, but it works like before I made changes while reducing the width further. U can checkout without deployment. BTW it is fine in desktop view and mobile view after your suggestions.

Comment (MeghalBisht) [reply app/styles/partials/event-card.scss:56]: **current -**
Screenshot from 2021-02-04 19-44-34

**after your PR -** 
Screenshot from 2021-02-04 19-44-10

The issue is about the overflowing text which is still not resolved.

Comment (afzal442) [reply app/styles/partials/event-card.scss:56]: Sorry for this, jst asking, this error happens when this page is in responsive mode but it's been fixed in both desktop and mobile view. If u want me to resolve that above issue, then there will be another error like height gets increased in order card causing extra wide space in the image area in desktop view.

**Comment by afzal442** [comment]
@mariobehling @iamareebjamal sir, I have fixed this issue in both desktop and mobile view as per the issue's demand. Plz review it.

File: app/styles/partials/event-card.scss (hunk 2 above)
Comment (afzal442) [reply app/styles/partials/event-card.scss:56]: @MeghalBisht sorry for the misunderstanding. I got you now that it is to make further changes in responsive mode also.

**Review by maze-runnar — APPROVED** [review]

File: app/styles/partials/event-card.scss [hunk 3]
```diff
@@ -54,4 +54,4 @@
...
-}
+  }
```
Comment (iamareebjamal) [inline app/styles/partials/event-card.scss:57]:
```suggestion
}
```

**Pull request merged**